lib/components/viewers/nearby/vehicle-rent.tsx

```diff
@@ -106,7 +95,15 @@ const Vehicle = ({
       <CardHeader>
         <CardTitle>
           <IconWithText
-            icon={<StationIcon companyLabel={companyLabel} vehicle={vehicle} />}
+            icon={
+              <CompanyIcon
+                company={vehicle.network}
+                fallbackContent={<span>{getVehicleIcon(formFactor)}</span>}
```

Comment:
what's the span for?

Reply:
It was there when you wrote this component originally! Seems that it is not needed, so I removed it in https://github.com/opentripplanner/otp-react-redux/pull/1347/commits/31a4c8ded0be9772d5e25b0a6cdbb28ec10f2dc6.